src/vector.rs

```diff
@@ -155,6 +155,11 @@ impl Vector {
             y: rotated_y,
         }
     }
+
+    /// Negates the vector
```

Comment:
", returning a vector with the same magnitude pointing in the opposite direction"

Reply:
ed7c0d2